event/models.py

```diff
@@ -8,6 +10,8 @@ class Event(models.Model):
     start_datetime = models.DateTimeField()
     end_datetime = models.DateTimeField()
     organisation = models.ForeignKey(Organisation, on_delete=models.CASCADE)
+    logo = models.FileField(upload_to='events/', blank=False, null=True,
```

Comment:
event gets the cover, organisation gets the logo - you may need to delete the current migrations for it to work properly

Reply:
Fixed.